Release Notes/2018/2018-05-07-cloud-platform-release-notes.md

```diff
@@ -1,16 +1,54 @@
 {{{
 "title": "Cloud Platform - Release Notes: May 7, 2018",
-"date": "05-07-2018",
-"author": "David O'Brien",
+"date": "05-08-2018",
+"author": "Thomas Broadwell",
 "attachments": [],
 "contentIsHTML": false
 }}}
 
 ### Enhancements (1)
 
-##### Cloud Application Manager
+#### Cloud Application Manager
+
+##### Managed Services Anywhere
+
+We are excited to launch few major enhancements to our Managed Services Anywhere product. As you are aware we have launched our initial set of Managed Services on AWS in February of 2017 and expanded to Azure in April of 2017. We are now announcing additional enhancements to our Managed Services Offering.
+
+###### Supported Platforms
+
+Besides AWS & Azure we are now expanding our supported platforms to include CenturyLink Private Cloud (Dedicated Cloud Compute – Foundation). CenturyLink Private Cloud (DCC-F) users can now signup for Cloud Application Manager and enable Managed Services and get the same features that they would get on AWS & Azure.
+
+###### Improved set of fully supported services:
+
+Our initial Managed Services offering included Managed Operating System of Microsoft Windows and RedHat Enterprise Linux flavors. Our expanded set of services now include:
+
+A) Managed Operating System of Microsoft Windows and RedHat Enterprise Linux
+
+B) Managed Applications (Microsoft SQL Server, Microsoft IIS Web Server, Oracle MySQL Enterprise Edition, Apache Web Server, Apache Tomcat)
+
+C) AWS Services (Amazon EC2, VPC, Relational Database Services, EBS, S3, ELB, Route 53, ElastiCache, CloudFront, Management Tools, Cloud Formation, Direct Connect, Cloud Watch, Cloud Trail, Trusted Advisor, Cert Manager, Organizations, Auto Scale, Directory Services, IAM)
+
+D) Azure Services (Virtual Machines, Virtual Network, VPN Gateway, Security Groups, ExpressRoute, IP Addresses, Blob Storage, Queue Storage, Disk Storage
+
+E) DCC-Foundation Edge Gateway & VMware vCloud Director service features
+
+###### Service enablement
+Going forward customers can enable Managed Services by "Provider" instead of enabling by instance. Provider is a representation of an Account in the downstream cloud platform. Once Managed Services are enabled on a Provider, CenturyLink will provide full support on the services listed above.
+
+###### Billing
+Going forward Managed Services charge is calculated based on the spend of the Provider on which it is enabled unlike the per instance/per hour model. Managed Services are charged on a graduated spend.
+
+Graduated Pricing Tiers:
+
+$0 - $50k = 45%
```

Comment:
probably a table or line breaks needed here. its making it a single line in preview.